### ISSUE TITLE
reconnect and send after close method; sender doesn't have a "closed" state

### ISSUE THREAD

**Issue by dehatanes** [fluent/fluent-logger-python#110]
I can't shutdown the client with the close() method.
I am using python 2.7 on jupyter notebook and structured the code as your example:

```
from fluent import sender
from fluent import event

host_name = (my_host_url)
port = 24224
logger = sender.FluentSender('app', host=host_name, port=port)

if not logger.emit("test", {'message': 'potato'}):
       print(logger.last_error)
       logger.clear_last_error()

logger.close()

if not logger.emit("test", {'message': 'Oops'}):
       print(logger.last_error)
       logger.clear_last_error()
```

The second log with the "Oops" message is emitted as nothing happened :(

I tried with FluentSender Interface and  Event-Based Interface and both did the same thing.
Did I misunderstood the purpose of closing the client (I supposed it would cut the connection with the client and prevent me of emitting more logs) or am I forgetting to do something when I try to close it?
 
Thanks,
Débora.

**Comment by arcivanov** [comment]
What version of fluent-logger are you using?

**Comment by arcivanov** [comment]
Please try the latest master @dehatanes

**Comment by arcivanov** [comment]
Your logger.close() happens before the last emit. Why do you expect the 'Oops' to go anywhere?

**Comment by dehatanes** [comment]
What version of fluent-logger are you using? Please try the latest master
fluent-logger==0.6.0

              Your logger.close() happens before the last emit. Why do you expect the 'Oops' to go anywhere?
Exactly, it shouldn't go anywhere (I guess), but it is going. I don't know if it is a bug, but my conexion with the client is apparently not being closed.

Thanks, @arcivanov

**Comment by arcivanov** [comment]
I could be a bug. I'm trying to release 0.8.0 right now. Please try the latest master `pip install -U git+git://github.com/fluent/fluent-logger-python` and see if it helps.

**Comment by dehatanes** [comment]
Thanks, @arcivanov 
I just installed the latest release 0.8.0 as you told me and tried it, but the same bug happened and the two logs were emitted :(
Is there anything I can do or send you that can help you solve the problem?

**Comment by arcivanov** [comment]
I'll run some tests to see what's up.

**Comment by arcivanov** [comment]
Ah, I just realized what the problem is... There isn't really "closing" of the sender right now as it simply reconnects once it receives another message. I'll fix it.

**Comment by arcivanov** [comment]
Relevant code:
 https://github.com/fluent/fluent-logger-python/blob/master/fluent/sender.py#L112
 https://github.com/fluent/fluent-logger-python/blob/master/fluent/sender.py#L165

**Comment by dehatanes** [comment]
Great, @arcivanov , thank you! :)

**Comment by arcivanov** [comment]
@dehatanes the fix is contained in version 0.9.0
